### ISSUE TITLE
Mica for MacOS

### ISSUE THREAD

**Issue by EdwardLangdon** [bmFtZQ/edge-frfox#34]
I think Mica/Translucent effect for MacOS can be acheived by using this CSS - https://github.com/zvuc/firefox-macos-native-tabbar

I would like to make a PR but I have a custom fork for my own customization. So, I don't want to mess it up.

**Comment by bmFtZQ** [comment]
It looks like this is being removed in Firefox 103 unfortunately, it has already been removed in Firefox Nightly:

https://github.com/mozilla/gecko-dev/commit/f9b1d27b1704678516f0fbfcea4fb5bcc1344fe7

If I were to add it now, it would only really work until the end of July when Firefox 103 releases. This property stopped being used when Firefox transitioned to the Proton design, so I guess it had no real reason to stick around after that.

**Comment by EdwardLangdon** [comment]
So, The Main project is also dead after FF103 or are there any other ways to implement

**Comment by bmFtZQ** [comment]
Out of curiosity I tried `appearance: -moz-mac-source-list` since I knew it also produced the vibrancy effect, and it worked. Unsure if there are any side effects though, as this appearance value is only used for sidebars.

Light window (Nightly 103.0a1 (2022-06-13)):
![Firefox window showing vibrancy behind tabs (light mode)](https://user-images.githubusercontent.com/62812711/173569911-a5354d03-7968-4aab-a946-5a49aeb28d5c.png)

Dark window (Nightly 103.0a1 (2022-06-13)):
![Firefox window showing vibrancy behind tabs (dark mode)](https://user-images.githubusercontent.com/62812711/173570039-3358773c-c5f4-41e2-bf3e-5a8040baa3e2.png)

But I wouldn't be surprised if this was removed at some later date as it doesn't really fit with the Proton design (but there are no plans to do so afaik):
![Firefox window showing the Proton design and sidebar](https://user-images.githubusercontent.com/62812711/173571093-aa2ae42e-f6aa-48bc-bdc5-aaa33988a440.png)

I will probably add this with a warning that it hasn't been properly tested.
